scripts/nlp_language_modeling/preprocess_data_for_megatron.py

```diff
@@ -221,10 +222,16 @@ def get_args():
         help='What tokenizer library to use.',
     )
     group.add_argument(
-        '--tokenizer-type', type=str, default=None, help='What type of tokenizer to use.',
+        '--tokenizer-type',
+        type=str,
+        default=None,
+        help='What type of tokenizer to use.',
```

Comment:
Seems that there's no code change here. Could you avoid formatting this line?

Reply:
i think this is because of the new isort and black formatting we use in nemo. All PRs are like this now..

Reply:
Yes I don't think I made this change on my end. Should I still revert to the old formatting or will the tests fail if I try to revert?

Reply:
you can't revert this. the auto commit will change it back